tests/peft_integration/test_peft_integration.py

```diff
@@ -265,17 +265,31 @@ def test_peft_add_multi_adapter(self):
                 _ = model.generate(input_ids=dummy_input)
 
                 model.set_adapter("default")
-                self.assertTrue(model.active_adapter() == "default")
+                self.assertTrue(model.active_adapter() == ["default"])
+                self.assertTrue(model.active_adapter(return_multi_adapters=False) == "default")
```

Comment:
This is a breaking change; are we ok with that breaking change?

Reply:
We can set the default `return_multi_adapters` to `True`, but there will be still the case of users that used the main branch of PEFT for multiple active adapters that will face a behaviour which is going to be different whether they use 2 different versions of transformers - cc @BenjaminBossan also what do you think

Reply:
So I think the most common case would be users with only a single active adapter, as multiple active adapters just landed on PEFT main and have not been advertised yet. Therefore, I would suggest to keep BC for those users and accept that maybe a tiny minority who uses multiple active adapters with the current transformers version will get a breaking change when they upgrade transformers.